### PR TITLE
Added Link attribute in Embedder and LLM python files

### PR DESCRIPTION
**Pull request by mustafaazad03**
# Description

In this PR, Issue addressed #530 where I added specific links in embedder and LLM file to point the specific embedder.

Related to issue #530 

## Type of change

- [x] Added specific links in the embedder file
- [x] Added specific link attribute in LLM file

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: core/Dockerfile [hunk 1]
```diff
@@ -14,7 +14,7 @@ RUN apt-get -y update && apt-get install -y curl build-essential fastjar libmagi
 
 ### ADMIN (static build) ###
 WORKDIR /admin
-RUN curl -sL https://github.com/cheshire-cat-ai/admin-vue/releases/download/Admin/develop.zip | jar -xv
+RUN curl -sL https://github.com/cheshire-cat-ai/admin-vue/releases/download/Admin/release.zip | jar -xv
```
Comment (nicola-corbellini) [inline core/Dockerfile:17]: Please, this should not be changed

**Comment by zAlweNy26** [comment]
Please don't change the dockerfile.

**Review by nicola-corbellini — CHANGES_REQUESTED** [review]

**Comment by Pingdred** [comment]
You have created the branch for the PR from main, please close and reopen it by making the PR from develop to develop, thanks.

File: core/Dockerfile (hunk 1 above)
Comment (mustafaazad03) [reply core/Dockerfile]: sorry for this, I change the file.

**Review by nicola-corbellini — COMMENTED** [review]
The code is good now, but still you are bringing the commits from the `main` branch. As @Pingdred suggested, you should close this PR, create a new branch starting from `develop`, do your changes and open a new PR. Thank you

**Comment by pieroit** [comment]
@mustafaazad03 plese redo the PR from `develop` branch, otherwise we close this

**Comment by mustafaazad03** [comment]
Ok sure

**Pull request closed without merging**